src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -384,9 +385,9 @@ export const BaseAnalyses = ({
   //We reference the analyses from `useAnalysisStore`, and on change, we decorate them from `AnalysisFile[]` to `DisplayAnalysisFile[]` and update state history
   useEffect(() => {
     const rawAnalyses: AnalysisFile[] = loadedState.status !== 'None' && loadedState.state !== null ? loadedState.state : []
-    const decoratedAnalyses = decorateAnalysisFiles(rawAnalyses)
-    setAnalyses(decoratedAnalyses)
-    StateHistory.update({ analyses: decoratedAnalyses, sortOrder, filter })
+    const sortedAnalyses = _.reverse(_.sortBy(tableFields.lastModified, rawAnalyses))
```

Comment:
the only thing I stole from the decorateAnalysisFiles function was the sorting bit that's here now